### ISSUE TITLE
[map] implement polygon search

### ISSUE THREAD

**Issue by bakeiro** [Couchers-org/couchers#4011]
**Current behavior:**

I open the map and search for Thailand, I only get 2 hosts there, I delete the filter, then I get that more than 100 people in Thailand (pretty big issue 🙃) [video attached]

**Desired behavior:**  
I search for Thailand and I get the real number of users that are in Thailand

https://github.com/Couchers-org/couchers/assets/16777161/cd56fd52-f42a-4863-9530-a22426dd9a1e

**Comment by bakeiro** [comment]
from the frontend side
![image](https://github.com/Couchers-org/couchers/assets/16777161/22b346a7-5016-4bde-9300-93acc7051a5f)

**Comment by aapeliv** [comment]
What do you mean by this? Who will define the polygons, what will it be used for?

**Comment by bakeiro** [comment]
to be defined, didn't define the task yet, steps needs to be defined yet, basically to make the search map usable/complete

**Comment by bakeiro** [comment]
### Implementation:

[frontend](https://github.com/Couchers-org/couchers/issues/4082) Get the geometries from nominatim, then send them to the backend

backend: [this ticket] Once it receives the geometries from the frontend, search the users inside this geometry only

**Note:**  
Frontend only will send the geometries to the backend if the area it's an non-region (more info in the frontend ticket)

**Comment by bakeiro** [comment]
I don't think we need to do this, since we have the viewbox search now...

**Comment by nabramow** [comment]
@bakeiro looks like this should be marked as closed is that right?

**Comment by bakeiro** [comment]
@nabramow yes & no, I want to double check it myself first (I didn't have time for that 😅)

**Comment by bakeiro** [comment]
the status is `paused` because of this 😅

**Comment by nabramow** [comment]
> the status is `paused` because of this 😅

Sorry I accidentally closed it that was a mistake!! Reopened lol but in case you get the email and are like wtf.
